etl_tests/models/test_document_tuple.py

```diff
@@ -1,23 +1,27 @@
 from langchain.docstore.document import Document
 
-from etl.models import wikipedia
+from etl.models import DocumentTuple, wikipedia
 from etl.models.types import EnrichmentType, RecordType
-from etl.utils import create_documents
 
 
-def test_create_documents(
+def test_document_tuple_from_records(
     document_of_article_with_summary: Document,
     tuple_of_articles_with_summaries: tuple[wikipedia.Article, ...],
     record_type: RecordType,
     enrichment_type: EnrichmentType,
 ) -> None:
-    """Test that create_documents successfully creates and returns a Document."""
+    """
+
+    Test that DocumentTuple.from_records successfully takes in a tuple of Records,
+    and returns a new object that contains a tuple of Documents.
+
+    """
 
     assert (
-        create_documents(
+        DocumentTuple.from_records(
             records=tuple_of_articles_with_summaries,
             record_type=record_type,
```

Comment:
I'm a bit puzzled by passing around record_type, since

1. Everything is in Python, not passing between languages. You can use `instanceof` with end-to-end types.
2. There's only one value of `record_type`.